code/web/src/modules/subscription/api/state.js

```diff
@@ -92,7 +92,7 @@ export const subscriptionsByUser = (state = subscriptionsByUserInitialState, act
       return {
         ...state,
         isLoading: false,
-        dateAvailable: action.date
+        nextDeliveryDate: action.date
```

Comment:
Thanks for updating this key in the `subscriptionsByUser` so that it matches the backend.